common/src/main/java/com/redhat/cloud/notifications/models/WebhookProperties.java

```diff
@@ -17,7 +20,11 @@
 @Table(name = "endpoint_webhooks")
 @JsonNaming(SnakeCaseStrategy.class)
 public class WebhookProperties extends EndpointProperties {
+
+    @AllowedScheme
     @NotNull
+    @NotPrivateHostname
+    @WellFormedUrl
```

Comment:
I am going to check that right now and post the results in the ticket.

Reply:
I have updated the Jira ticket with details of the issue, but I want to leave notice here too. Basically we have got a few endpoints that don't comply with this new annotation. My idea was to disable all of them, because there are a few endpoints that are still enabled —and if they were used, they would be probably auto-disabled by our "disable after N retries" policy—, which would make the users that are interested in using that endpoint to reintroduce a valid URL when editing the endpoint.